frontend/src/pages/modelRegistry/screens/RegisteredModels/RegisteredModelTableRow.tsx

```diff
@@ -32,19 +33,21 @@ const RegisteredModelTableRow: React.FC<RegisteredModelTableRowProps> = ({
   const [isRestoreModalOpen, setIsRestoreModalOpen] = React.useState(false);
   const rmUrl = registeredModelUrl(rm.id, preferredModelRegistry?.metadata.name);
 
-  const actions = isArchiveRow
-    ? [
-        {
+  const actions = [
+    {
+      title: 'View details',
```

Comment:
Can we add a Cypress test for `View details`(maybe tweak the existing tests to add this option)?